demos/browser/app/meetingV2/videofilter/CircularCut.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  CanvasVideoFrameBuffer,
+  VideoFrameBuffer,
+  VideoFrameProcessor,
+} from '../../../../../src/index';
+
+/**
+ * [[CircularCut]] is an implementation of {@link VideoFrameProcessor} for demonstration purpose.
+ * It updates the first {@link VideoFrameBuffer} from the input array and clip the whole frame to a circle.
+ */
+export default class CircularCut implements VideoFrameProcessor {
+  private targetCanvas: HTMLCanvasElement = document.createElement('canvas') as HTMLCanvasElement;
+  private targetCanvasCtx: CanvasRenderingContext2D = this.targetCanvas.getContext(
+    '2d'
+  ) as CanvasRenderingContext2D;
+  private canvasVideoFrameBuffer = new CanvasVideoFrameBuffer(this.targetCanvas);
+  private sourceWidth: number = 0;
+  private sourceHeight: number = 0;
+
+  constructor(private radius: number = 150) {}
+
+  name(): string {
+    return 'CircularCut';
+  }
+
```

Comment:
I would remove this and see how it changes your demo app.

Reply:
updated and demo do not call this function.